src/components/common/notification-center.svelte

```diff
@@ -0,0 +1,51 @@
+<script>
+  import { onMount } from 'svelte';
+  import Dropdown from 'ui/dropdown.svelte';
+  import Button from 'ui/button.svelte';
+  import * as api from '@/utils/api.js';
+
+  export let notifications = null;
+  $: unread = notifications != null && notifications.some(notification => !notification.is_read);
+
+  onMount(async () => {
+    try {
+      let resp = await api.get('/notifications');
+      if (resp.status === 200) {
+        notifications = await resp.json();
+      }
+    } catch (e) { console.error(e); }
+  });
+</script>
+
+<Dropdown
+  dropdownclass="notification-center"
+  handleclass="round"
+  chevron={false}
+  nowrap
+  let:toggle={toggle}
+>
+  <div slot="label" class="badge" class:hidden={!unread}>
+    <svg src="images/icons/bell.svg" class="icon" />
```

Comment:
It is really better to load from root like `/images/icons/...`, because in the nested pages it will look for, for example, '/item/3/images/icons/...`

Reply:
It is really better not to alias routes all over the place, but as for icons, I prefer to keep them slash-less